docs/source/index.rst

```diff
@@ -2,31 +2,28 @@
 JupyterHub
 ==========
 
-`JupyterHub`_ is the best way to serve `Jupyter notebook`_ for multiple users.
-It can be used in a class of students, a corporate data science group or scientific
+`JupyterHub`_ is the best way to serve `Jupyter notebook`_ for multiple users. The distributed Jupyter Notebook environment allows 
```

Comment:
Can we still remove this clause? I don't think the text before `it` on the next line is clarifying (or accurate).
Maybe: "Because JupyterHub manages a separate Jupyter environment for each user, it can be used..."